src/components/picker/index.js

```diff
@@ -160,6 +160,9 @@ class Picker extends PureComponent {
     // if (props.useNativePicker && _.isPlainObject(props.value)) {
```

Comment:
Please remove the commented code and the relevant TODO at the top of the file to avoid confusion.

Reply:
Right! 
Removed